integration-tests/src/main/java/pl/touk/widerest/base/ApiTestBase.java

```diff
@@ -73,6 +73,11 @@
 
     protected CatalogPreparer givenCatalog;
 
+    @Resource
+    protected DefaultCurrencyExpectations thenDefaultCurrency;
```

Comment:
\[Checkstyle\] ERROR: Variable 'thenDefaultCurrency' must be private and have accessor methods\.